src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/message.format/response.with.headers/client/nukleus.rpt

```diff
@@ -24,7 +24,7 @@ property targetInputEst ${nuklei.streams("http", "target")}
 property sourceInputEst ${nuklei.streams("source", "http#target")}
 
 property newTargetOutputId ${nuklei:newStreamId()}
-property newSourceInputEstId ${nuklei:newStreamId()}
+property sourceInputEstId ${nuklei:newStreamId()}
```

Comment:
The general naming strategy is to use `newXXX` if we generate via property section, or just `xxx` if we capture from the script body.

This one seems not to follow the naming convention.